soc/spid.c

```diff
@@ -108,8 +108,7 @@ struct pollfd fds[N_POLLFDS];
 int usbd_sock_fd;
 struct sockaddr_un usbd_sock_addr;
 void delay() {
-    volatile int i = 1000;
-    while(i--);
```

Comment:
> Was that before I fixed the kernel bug making timing unpredictable?

I'm not sure, I can't find a record of your kernel fix.

>  But I'd rather have it slow and working than fast and broken and it can be optimized later.

Without an extensive test bench, this the best we can do for now.